auto-deploy/deploy-haku.sh

```diff
@@ -30,11 +30,13 @@ for file in ${required_source_files}; do
 	fi
 done
 
+transfer_file_list+=" src"
+
 echo ${transfer_file_list}
 ssh -i ${deployment_key} ${deployment_user}@${deployment_host} << EOF
 [ -d ${deployment_path} ] && echo "skipping: ${deployment_path} exists" || mkdir -v ${deployment_path}
 EOF
-scp -i ${deployment_key} ${transfer_file_list} ${deployment_user}@${deployment_host}:${deployment_path}
+scp -ir ${deployment_key} ${transfer_file_list} ${deployment_user}@${deployment_host}:${deployment_path}
```

Comment:
The is not valid syntax, see my previous comment on requirements to scp.

-i requires input which is the deployment_key. When you pass in r after i, "i" thinks that "r" the deployment key. This is a common issue when programs use the getopt function from C.

Reply:
TIL